import-export-cli/integration/testUtils.go

```diff
@@ -320,6 +330,17 @@ func importAppPreserveOwner(t *testing.T, sourceEnv string, app *apim.Applicatio
 	return output, err
 }
 
+func importAppPreserveOwnerAndUpdate(t *testing.T, sourceEnv string, app *apim.Application, client *apim.Client) (string, error) {
+	fileName := base.GetApplicationArchiveFilePath(t, sourceEnv, app.Name, app.Owner)
+	output, err := base.Execute(t, "import-app", "--preserveOwner=true", "--update=true", "-f", fileName, "-e", client.EnvName, "-k", "--verbose")
+
+	t.Cleanup(func() {
+		client.DeleteApplicationByName(app.Name)
+	})
```

Comment:
This t.Cleanup() is not required since there is already one registered in the AddApplication function.